Dockerfile.jupyter

```diff
@@ -107,4 +107,4 @@ RUN mv notebooks/*.ipynb .
 RUN find . -name \*.ipynb -exec jupyter trust \{\} \;
 
 EXPOSE 8888
-ENTRYPOINT jupyter lab --ip sysmljupyter --port 8888
+ENTRYPOINT ["jupyter", "lab", "--ip", "sysmljupyter", "--port", "8888"]
```

Comment:
You learn something new every day.

Reply:
Indeed. This particular change was suggested by Docker when building; it's not _necessary_, but it results in OS signals being passed to the entry point process rather than to a shell starting the entry point process.